helm_deploy/hmpps-manage-offences-api/values.yaml

```diff
@@ -13,7 +13,8 @@ generic-service:
     enabled: true
     host: app-hostname.local    # override per environment
     tlsSecretName: manage-offences-api-cert
-    contextColour: green
+    v1_2_enabled: true
+    v0_47_enabled: false
```

Comment:
Due to manage offences not being properly live yet we can accept the small amount of downtime that turning 0.47 controller off gives. (true/true would give zero downtime)